sops.go

```diff
@@ -43,6 +43,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/hashicorp/vault/shamir"
```

Comment:
it's a very small library inside a gigantic project. I think we should duplicate it.